### PR TITLE
725 correction of formula for susceptibles in initialization in IDE-SECIR

### LINKED ISSUE #725: Correction of formula for Susceptibles in initialization in IDE-SECIR
In the formula to compute the number of Susceptibles at time 0 we use the formula from the discretization that uses the flows at a certain time point. However, in the implementation we can only use the flows over a time interval. Hence, the formula needs to be adapted, i.e. we have to divide by the size of the time step dt, see the Overleaf document.

### PR DESCRIPTION
**Pull request by annawendler**
# Changes and Information

In the formula to compute the number of Susceptibles at time 0 we used the formula from the discretization that uses the flows at a certain time point. However, in the implementation we can only use the flows over a time interval. Hence, the formula was adapted, i.e. we divided by the size of the time step dt, see also the Overleaf document.

## Merge Request - Guideline Checklist

Please check our [git workflow](https://github.com/DLR-SC/memilio/wiki/git-workflow). Use the **draft** feature if the Pull Request is not yet ready to review.

### Checks by code author

- [x] Every addressed issue is linked (use the "Closes #ISSUE" keyword below)
- [x] New code adheres to [coding guidelines](https://github.com/DLR-SC/memilio/wiki/Coding-guidelines)
- [x] No large data files have been added (files should in sum not exceed 100 KB, avoid PDFs, Word docs, etc.)
- [x] Tests are added for new functionality and a local test run was successful
- [x] Appropriate **documentation** for new functionality has been added (Doxygen in the code and Markdown files if necessary)
- [x] Proper attention to licenses, especially no new third-party software with conflicting license has been added

### Checks by code reviewer(s)

- [x] Corresponding issue(s) is/are linked and addressed
- [x] Code is clean of development artifacts (no deactivated or commented code lines, no debugging printouts, etc.)
- [x] Appropriate **unit tests** have been added, CI passes and code coverage is acceptable (did not decrease)
- [x] No large data files added in the whole history of commits(files should in sum not exceed 100 KB, avoid PDFs, Word docs, etc.)

Closes #725

### CODE REVIEW

File: cpp/tests/test_ide_secir.cpp [hunk 1]
```diff
@@ -154,23 +154,23 @@ TEST_F(ModelTestIdeSecir, compareWithPreviousRunTransitions)
     }
 }
 
-// check results of our simulation with an example calculated by hand
-// for example see Overleaf document
+// Check results of our simulation with an example calculated by hand,
+// for calculations see Overleaf document.
```
Comment (mknaranja) [inline cpp/tests/test_ide_secir.cpp:158]:
```suggestion
// for calculations see Overleaf document.
// TODO: Add link to material when published.
```
```

File: cpp/tests/test_ide_secir.cpp [hunk 2]
```diff
@@ -426,26 +426,27 @@ TEST(IdeSecir, checkProportionRecoveredDeath)
...
     mio::ExponentialDecay expdecay(4.0);
     mio::StateAgeFunctionWrapper delaydistribution(expdecay);
     std::vector<mio::StateAgeFunctionWrapper> vec_delaydistrib(num_transitions, delaydistribution);
     vec_delaydistrib[(int)mio::isecir::InfectionTransition::InfectedCriticalToRecovered].set_parameter(3.0);
     model.parameters.set<mio::isecir::TransitionDistributions>(vec_delaydistrib);
 
-    // Set probabilities so that all individuals go from Susceptible to InfectedCritical with probability 1, from there they move
-    // to Recovered or Dead with probability 0.5, respectively.
+    // Set probabilities so that all individuals go from Susceptible to InfectedCritical with probability 1,
+    // from there they move to Recovered or Dead with probability 0.5, respectively.
```
Comment (mknaranja) [inline cpp/tests/test_ide_secir.cpp:449]: I think this should be 0.4 and 0.6, shouldnt it?

Comment (mknaranja) [inline cpp/tests/test_ide_secir.cpp:449]:
```suggestion
    // from there they move to Recovered or Dead with probability 0.4 and 0.6, respectively.
```

File: cpp/tests/test_ide_secir.cpp [hunk 3]
```diff
@@ -526,27 +527,28 @@ TEST(IdeSecir, compareEquilibria)
...
 
     // Set working parameters.
-    // Here we set the support_max for the TransitionDistribution%s differently for both models.
-    // In both models, all TransitionDistribution%s are SmootherCosineFunctions
+    // Here the maximum support for the TransitionDistribution%s is set differently for each model
+    // In both models, all TransitionDistribution%s are SmootherCosine.
 
-    // For model
-    // All TransitionDistribution%s have funcparam=2
+    // For model.
```
Comment (mknaranja) [inline cpp/tests/test_ide_secir.cpp:533]:
```suggestion
    // For the Model model.
```

File: cpp/tests/test_ide_secir.cpp [hunk 4]
```diff
@@ -526,27 +527,28 @@ TEST(IdeSecir, compareEquilibria)
...
+    // All TransitionDistribution%s have parameter=2.
     mio::SmootherCosine smoothcos(2.0);
     mio::StateAgeFunctionWrapper delaydistribution(smoothcos);
     std::vector<mio::StateAgeFunctionWrapper> vec_delaydistrib(num_transitions, delaydistribution);
     model.parameters.set<mio::isecir::TransitionDistributions>(vec_delaydistrib);
 
     // For model2
```
Comment (mknaranja) [inline cpp/tests/test_ide_secir.cpp:540]:
```suggestion
    // For the Model model2.
```

File: cpp/tests/test_ide_secir.cpp [hunk 5]
```diff
@@ -221,22 +220,23 @@ TEST(IdeSecir, checkSimulationFunctions)
     mio::TimeSeries<ScalarType> secihurd_simulated    = sim.get_result();
     mio::TimeSeries<ScalarType> transitions_simulated = sim.get_transitions();
 
-    // Define vectors with values from example (calculated by hand, see Overleaf document)
+    // Define vectors for compartments and transitions with values from example
+    // (calculated by hand, see Overleaf document).
```
[the remark this answers is not in the document]
Comment (annawendler) [reply cpp/tests/test_ide_secir.cpp:224]:
```suggestion
    // (calculated by hand, see internal Overleaf document).
```

**Review by mknaranja — APPROVED** [review]

**Pull request merged**